configs/14.0/packages/glibc/stage_2

```diff
@@ -83,6 +83,11 @@ atcfg_pre_configure() {
 atcfg_configure() {
 	local base_target=$(find_build_target ${AT_BIT_SIZE})
 
+	if [[ ("${distro_fm}" == "redhat") ||
+		("${distro_fm}" == "suse" )  ||
+		("${DISTRO_ID}" == "ubuntu-20") ]]; then
+		disable_crypt=yes
+	fi
```

Comment:
This part doesn't have a comment.
Why is this needed?
Should this actually test if `/usr/include/crypt.h` defines `XCRYPT_VERSION_STR` ? That would say if the libxcrypt development files are available in the OS.
But be careful because the test I mentioned is only valid for native builds.  Cross builds should not test it.

Reply:
This test doesn't work in case of Rhel8. 
In this release, libxcrypt´s version is 4.11,and  `/usr/include/crypt.h` doesn't have `XCRYPT_VERSION_STR` definition.